Makefile

```diff
@@ -239,3 +239,40 @@ ready-pr: lint manifests resource-docs generate-go-client
 upgrade-dcl:
 	go get github.com/GoogleCloudPlatform/declarative-resource-client-library
 	make ensure
+
+# Build all binaries
+all-binary: cmd-config-connector deletiondefender cmd-manager recorder unmanageddetector webhook gke-addon-poststart operator-cmd-manager
+.PHONY: all-binary cmd-config-connector deletiondefender cmd-manager recorder unmanageddetector webhook gke-addon-poststart operator-cmd-manager
```

Comment:
Generally better to leave 1 .PHONY directive per target, rather than grouping them this way. Makes it easier to keep the targets clean and clean.